### PR TITLE
Backgrounds

### PR DESCRIPTION
**Pull request by sophieheb**

### CODE REVIEW

File: src/pages/_app.js [hunk 1]
```diff
@@ -120,7 +128,9 @@ function App({ Component, pageProps }) {
                 flexDirection="column"
                 flexGrow={1}
                 overflow="scroll"
-                className={showBackground ? classes.background : null}
+                className={`${background === 'none' ? classes.backgroundNone : ''} 
+                ${background === 'blue' ? classes.backgroundBlue : ''}
+                ${background === 'peach' ? classes.backgroundPeach : ''}`}
```
Comment (jits) [inline src/pages/_app.js:133]: Potentially silly question; could you set the state to be the class name and then just do something like `className={backgroundClass}`?

**Review by jits — APPROVED** [review]
One suggestion, otherwise LG!

File: src/pages/_app.js (hunk 1 above)
Comment (sophieheb) [reply src/pages/_app.js:133]: Tried this before but with fresh eyes I think I might be able to do something! Let me try & get back to you

Comment (sophieheb) [reply src/pages/_app.js]: Made better!

**Pull request merged**